docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -40,11 +40,12 @@ Scheduler schedulerWithMetrics = Micrometer.timedScheduler(
 IMPORTANT: When wrapping a common `Scheduler` (eg. `Schedulers.single()`) or a `Scheduler` that is used in multiple places, only the `Runnable` tasks that are
 submitted through the wrapper instance returned by `Micrometer#timedScheduler` are going to be instrumented.
 
-See <<micrometer-details-timedScheduler>> for produced meters and associated default tags.
+See xref:metrics-details.adoc#micrometer-details-timedScheduler[Micrometer.timedScheduler()] for produced meters and associated default tags.
 
-// FIXME reactor-monitoring-demo won't be in sync with 3.5.0 anymore
+FIXME reactor-monitoring-demo won't be in sync with 3.5.0 anymore
```

Comment:
This bit is supposed to be hidden

Reply:
Fixed in https://github.com/reactor/reactor-core/pull/3806/commits/b2f925c2dac4632726f5452c615adc2fc108098d